### ISSUE TITLE
Lint warning about TrustAllX509TrustManager in TCSSLSocketFactory$1.class:0

### ISSUE THREAD

**Issue by cedric-rss** [TagCommander/Android#15]
Hello,

I am currently setting latest version of TrustCommander for my compagny app

```
# https://github.com/TagCommander/Android
tag_commander_core = "4.7.0"
tag_commander_privacy = "4.10.0"
tag_commander_TCIAB = "4.7.1"
tag_commander_SDK = "4.6.0"
```

And I got this lint warning in TCSSLSocketFactory$1.class:0 :

```
Insecure TLS/SSL trust manager `checkClientTrusted` is empty, which could cause insecure network traffic due to trusting arbitrary TLS/SSL certificates presented by peers This check looks for X509TrustManager implementations whose `checkServerTrusted` or `checkClientTrusted` methods do nothing (thus trusting any certificate chain) which could result in insecure network traffic caused by trusting arbitrary TLS/SSL certificates presented by peers.
```

Have you planned anything to fix this issue ?

**Comment by CommandersAct** [comment]
Hello !
This will be fixed in the v5 release incoming later this month.

**Comment by CommandersAct** [comment]
Should be fixed with TCCore 4.7.1!
